app/controllers/jobseekers/job_applications_controller.rb

```diff
@@ -26,12 +25,14 @@ def new
   end
 
   def create
-    new_job_application = current_jobseeker.job_applications.create(vacancy:)
-    redirect_to jobseekers_job_application_apply_path(new_job_application)
+    new_job_application = vacancy.build_job_application_for(current_jobseeker)
+    new_job_application.save!
```

Comment:
If every time we call this we end up doing a `build` and then a `save!` immediately after (I think we always do that, right?)... couldn't we just define a `create!` method instead?
```suggestion
    new_job_application = vacancy.create_job_application_for!(current_jobseeker)
```

Reply:
Sure, done 🫡